src/sentry/models/processingissue.py

```diff
@@ -79,16 +80,68 @@ def find_resolved(self, project_id, limit=100):
         eventstore.bind_nodes(rv, "data")
         return rv, has_more
 
+    @staticmethod
+    def is_release_newer(org_id, release, other_release):
+        if release == other_release:
+            return False
+
+        if other_release is None:
+            return True
```

Comment:
we return True here, `if release is None` too.

Reply:
Is it expected?

Reply:
We return `True` if `other_release is None`, otherwise we return `False`.